tests/go.mod

```diff
@@ -20,7 +21,7 @@ require (
 	github.com/mailru/easyjson v0.7.6 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.1.1 // indirect
-	github.com/spf13/pflag v1.0.5 // indirect
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9 // indirect
```

Comment:
Any specific reason to update go.mod?

Reply:
Not entirely sure what this is, TBH. It showed changed when I ran `make generate-changed-only` so committed it. Happy to pull it out if not relevant.